backend-server/src/api/data/dataAccess.ts

```diff
@@ -21,13 +21,25 @@ class DataAccess {
   interviewerCollectionName: string;
   rootCollection: CollectionReference;
 
-  constructor(db, rootCollectionName, interviewerCollectionName) {
+  constructor(
+    db: Firestore,
+    rootCollectionName: string,
+    interviewerCollectionName: string
+  ) {
     this.db = db;
     this.rootCollectionName = rootCollectionName;
     this.interviewerCollectionName = interviewerCollectionName;
     this.rootCollection = collection(db, rootCollectionName);
   }
 
+  async checkOrganizationExists(organization: string): Promise<boolean> {
+    const orgRef = await doc(this.rootCollection, organization);
+    const org = await getDoc(orgRef);
+
+    if (org.exists()) return true;
+    else return false;
```

Comment:
super nit: maybe we could write this as `return org.exists()`, not going to block this but just thinking it feels a little weird to check if a condition is true and then return true